functions/Test-DbaJobOwner.ps1

```diff
@@ -138,9 +138,10 @@ function Test-DbaJobOwner {
                 $row = [ordered]@{
                     Server       = $server.Name
                     Job          = $j.Name
+                    JobType      = if($j.CategoryID -eq 1){ "Remote" } else{ $j.JobType }
                     CurrentOwner = $j.OwnerLoginName
                     TargetOwner  = $Login
-                    OwnerMatch   = ($j.OwnerLoginName -eq $Login)
+                    OwnerMatch   = if($j.CategoryID -eq 1){ $true } else{ $j.OwnerLoginName -eq $Login }
```

Comment:
Space this out `if ($j.CategoryID -eq 1) { $true } else { $j.OwnerLoginName -eq $Login }`